pkg/schemas/generator.go

```diff
@@ -105,11 +137,125 @@ func (g Generator) Generate(ctx *genall.GenerationContext) error {
 			documents[documentName] = document
 		}
 		document.Definitions[context.definitionNameFor(documentName, typeIdent)] = typeSchema
+
+		// Generate a schema for types with "fybrik:validation:object" marker
+		info, knownInfo := parser.Types[typeIdent]
+		if knownInfo {
+			if info.Markers.Get(objectMarker.Name) != nil {
+				documentName := fmt.Sprintf("%s.json", info.Name)
```

Comment:
You may want to make this configurable in the marker so it would be possible to keep the same filenames as what exist today (e.g. fybrik_application vs FybrikApplication).

Reply:
Done.